# BasBees
Data and analyses accompanying 'Genetic differentiation at extreme latitudes in the socially plastic sweat bee Halictus rubicundus'

File: H.rub_raw_and_GenAlEx.xlsx
Processed MSAT scores (tab "D") and analyses from GenAlEx with definitions and explanations of all metrics used.
Tabs:
"D": Microsatellite scores (rounded to 0 dp) for all individuals included in the manuscript including 12 markers.
"PCoA": Principle component analyses including PC scores for all individual multilocus genotypes. Analyses conducted in GenAlEx 6.5 (also data.csv)
"Tot FstP": Fst scores and isolation by distance measures calculated in GenAlEx.
"APT": Allele frequencies and private allele frequencies for all populations calculated in GenAlEx.
"AGL": Allele frequencies and private allele frequencies for each microsatellite marker for each population calculated in GenAlEx.
"HFL": Summary statistics for every marker across all populations calculated in GenAlEx.
"NeiP": Nei genetic distances for all population pairs calculated in GenAlEx.
"FstP": Fst values for all population pairs calculated in GenAlEx.
"HW": Outcomes of tests for Hardy-Weinberg equilibrium for all genotypes at every locus by population calculated in GenAlEx.
"HWS": Summary of X^2 tests for Hardy-Weinberg equilibrium for all loci by population calculated in GenAlEx.
"WD": Consistency checks of raw data (missing data/scoring errors) calculated in GenAlEx.
"WDS": Consistency checks of (missing data/scoring errors) by population calculated in GenAlEx.

File: data.csv
Microsatellite scores (rounded to 0 dp) for all individuals included in the manuscript including 12 markers.

File: paramfile.txt 
Parameter details for running rarefied analysis (to calculate allele frequencies) analysis using ADZE. Links to datafile.txt.

File: datafile.txt
Raw MSAT scores for all individuals by population used for rarefied analysis using ADZE. 

File: rarefy_allelic_richeness.R
R code used to calculate standard and rarefied allelic richness 

File: w.o.locus 9[3].docx
Results table showing observed and expected heterozygosity when null allele (locus 9) removed. 


File: Supplementary information-BB.docx
Supplementary info about primers and primer sets used and additional methods from BM thesis.



